api/can_api_definition.py

```diff
@@ -122,6 +122,12 @@ class CANPredictionTimeseriesRow(pydantic.BaseModel):
         ...,
         description="Total ventilator capacity."
     )
+    Rt: float = pydantic.Field(
+        ..., description="Historical or Inferred Rt"
+    )
+    RtCI90: float = pydantic.Field(
+        ..., description="Rt standard deviation"
```

Comment:
should this be "R_t 90th percent confidence interval"?

Reply:
It's copied from the description for the field earlier in the file (approx line 30-35):
```
    RtCI90: float = pydantic.Field(
        ..., description="Rt standard deviation"
    )
```

Easy to switch to however we'd like to describe it, though.